Added envoy debug log configuration (#5608)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md

--component-log-level upstream:debug,connection:trace
```

For example, add the following line to the docker-compose.yaml file in the directory `<CHOREO-CONNECT_HOME>/docker-compose/<choreo-connect>/`.

```yaml
  router:
    environment:
      - TRAILING_ARGS=--log-level trace
```

> The following example configuration can log request headers and response headers, ext_authz information.

Reviewer: ```suggestion
The following example configuration can log request headers and response headers, and ext_authz information.
```

```yaml
  router:
    environment:
      - TRAILING_ARGS=TRAILING_ARGS=--component-log-level http:debug,http2:debug,conn_handler:debug,ext_authz:trace
```

> Following are different sections of a example log with the above log configuration. Log generated by invoking a API request with `/pet/findByStatus?status=sold` path in [Swagger Petstore](https://petstore.swagger.io/) API deployed to Choreo Connect. Different log sections are given in the order same as the order of they are logged to the console.

Reviewer: ```suggestion
The following are different sections of an example log with the above log configuration. This log is generated by invoking an API request with `/pet/findByStatus?status=sold` path in [Swagger Petstore](https://petstore.swagger.io/) API deployed to Choreo Connect. Different log sections are given in the order same as the order in which they are logged to the console.
```

    }
  }
  fields {
    key: "tokenType"
    value {
      string_value: "JWT"
    }
  }
  ...
  ...
```

Router to client response headers
```yaml
[2022-04-11 08:18:04.956][15][debug][http] [source/common/http/conn_manager_impl.cc:1467] [C19][S3973774370179828478] encoding headers via codec (end_stream=false):
':status', '200'
'date', 'Mon, 11 Apr 2022 08:18:04 GMT'
'content-type', 'application/json'
'access-control-allow-origin', '*'
'access-control-allow-methods', 'GET, POST, DELETE, PUT'
'access-control-allow-headers', 'Content-Type, api_key, Authorization'
'server', 'envoy'
'x-envoy-decorator-operation', 'localhost:^/v2/pet/findByStatus(\?([^/]+))?$'
```

> Please follow [Command line options]({{envoy_path}}/operations/cli) for more information.

Reviewer: ```suggestion
Follow [Command line options]({{envoy_path}}/operations/cli) for more information.
```

> Follow [Command line options]({{envoy_path}}/operations/cli) for more information.

Reviewer: Not sure what this link is. @pubudu538 is this how this is supposed to be?

Reviewer 2: Choreo Connect is built on top of Envoy. That is the reason for having this link.
